xray/analyze.py

```diff
@@ -84,6 +86,30 @@ def __init__(self, csv_name, pcap_name, count, test_type):
             fn(ax[i, 1])
 
         plt.show()
+        if save_output:
+            plt.savefig(test_path + ".png")
+
+        if ascii:
```

Comment:
Let's make this an if-else so we only show one of the two formats at the same time. And maybe it can be simplified since both branches would have some code in common

Reply:
Converted into an if-else case